src/utility/index.jsx

```diff
@@ -265,14 +272,24 @@ export const transformations = {
       </div>);
   },
   game_mode: (row, col, field) => (strings[`game_mode_${field}`]),
-  match_id_and_game_mode: (row, col, field) => (
-    <div>
-      <TableLink to={`/matches/${field}`}>{field}</TableLink>
-      <span className={subTextStyle.subText} style={{ display: 'block', marginTop: 1 }}>
-        {strings[`game_mode_${row.game_mode}`]} / {strings[`lobby_type_${row.lobby_type}`]}
-      </span>
-    </div>
-  ),
+  match_id_and_game_mode: (row, col, field) => {
+    const partySize = (partySize) => {
```

Comment:
An example 
![image](https://user-images.githubusercontent.com/6865538/28757811-9efa2830-7550-11e7-93f2-5b58f63abed5.png).

I couldn't find any matches that were in a party. Null would return nothing, solo would return like the above picture.